packages/react-components/src/components/TagInput/TagInput.stories.tsx

```diff
@@ -1,49 +1,68 @@
 import * as React from 'react';
-import { ComponentMeta } from '@storybook/react';
+import { ComponentMeta, Story } from '@storybook/react';
+
+import noop from '../../utils/noop';
+import { StoryDescriptor } from '../../stories/components/StoryDescriptor';
 
 import {
-  TagInput as TagInputComponent,
+  TagInput,
   TagInputProps,
-  EmailTagInput as EmailTagInputComponent,
+  EmailTagInput,
   EmailTagInputProps,
 } from './index';
 
+const placeholderText = 'Placeholder text';
+
 export default {
   title: 'Forms/Tag Input',
-  component: TagInputComponent,
+  component: TagInput,
   argTypes: {
     tags: {
       control: {
         disable: true,
       },
     },
   },
-} as ComponentMeta<typeof TagInputComponent>;
+} as ComponentMeta<typeof TagInput>;
 
-export const TagInput = ({ ...args }: TagInputProps): React.ReactElement => {
+export const DefaultTagInput: Story<TagInputProps> = ({
+  ...args
+}: TagInputProps) => {
   const [tags, setTags] = React.useState(['tag1', 'tag2']);
   return (
     <div>
-      <TagInputComponent {...args} tags={tags} onChange={setTags} />
+      <TagInput {...args} tags={tags} onChange={setTags} />
     </div>
   );
 };
 
-TagInput.args = {
-  placeholder: 'Tag input placeholder',
-} as TagInputProps;
+DefaultTagInput.storyName = 'TagInput';
+DefaultTagInput.args = {
+  placeholder: placeholderText,
+};
 
-export const EmailTagInput = ({
+export const DefaultEmailTagInput: Story<EmailTagInputProps> = ({
   ...args
-}: EmailTagInputProps): React.ReactElement => {
+}: EmailTagInputProps) => {
   const [tags, setTags] = React.useState(['one@test.com', 'two@test.com']);
   return (
     <div>
-      <EmailTagInputComponent {...args} tags={tags} onChange={setTags} />
+      <EmailTagInput {...args} tags={tags} onChange={setTags} />
     </div>
   );
 };
-
-EmailTagInput.args = {
+DefaultEmailTagInput.storyName = 'EmailTagInput';
```

Comment:
Shouldn't we use spaces in the story name?

Reply:
The default story name is simply the name of the component. I'm using this workaround to avoid the unnecessary rename of the imported component from `EmailTagInput` to `EmailTagInputComponent`. It's better this way as the `Docs` section in Storybook is not polluted with the `Component` suffix.